src/metabase/search/filter.clj

```diff
@@ -22,9 +22,12 @@
    :table-db-id         :database-id
    :verified            :verified})
 
+;; TODO dry this alias up with the index hydration code
+(def ^:private field-alias {:created-at :model-created-at})
```

Comment:
Without this we were confusing the time the underlying model was created, with the time when we first added it to the search index.

In future these will be roughly the same, but the error is quite large when you first upgrade... so I'm glad I caught this early 😓